MekHQ/src/mekhq/gui/WarehouseTab.java

```diff
@@ -670,8 +668,7 @@ protected void updateTechTarget() {
     }
 
     public void refreshTechsList() {
-        ArrayList<Person> techs = getCampaign().getTechs(true, null, false, false);
-        techsModel.setData(techs);
+        techsModel.setData(getCampaign().getTechs(true, null, true, false));
```

Comment:
A comment here as well would be nice. Also, if `getTechs(true, null, true, false)` is the new default, how about a descriptively named method on Campaign?

Reply:
It isn't the default, just used in these two places. Added the comment though